### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_MigratetoAzureSQL_MigrationAssessment_Keyboard:The focus on the checkbox is truncated and not clearly visible.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15773]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.4.7;#AINotInScope;#Keyboard

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till Migrate to azure sql and press enter.
5. Add azure account and tab till next button and press enter.
6. Select SQL server on azure virtual machine radio button and tab till view/select button and press enter.
7. Tab till database selection checkbox for test1 database under database table.

### Actual:
The focus on the checkbox is truncated and not clearly visible.

### Expected:
There should be a proper visible focus present on the checkbox.

### User Impact:
Low vision users may not be able to see the focus on the checkbox.

**Comment by Yashi1201** [comment]
Closing the bug as the issue is not repro
Version: 1.42.0-insider(user-setup)
![15773_issue fixed](https://user-images.githubusercontent.com/93374043/218384795-3c476178-b756-4080-a91c-a6f890edf345.gif)

**GithubTags**:#Closed;
